pootle/checks.py

```diff
@@ -547,3 +547,26 @@ def check_mysql_timezones(app_configs=None, **kwargs):
             if converted_now is None:
                 errors.append(missing_mysql_timezone_tables)
     return errors
+
+
+@checks.register()
+def check_mysql_collation(app_configs=None, **kwargs):
+    from django.db import connection
+
+    non_utf8_collation_and_character_set = checks.Critical(
+        _("MySQL charset and collation must support UTF8."),
+        hint=("Change charset and collation settings for you MySQL database"),
```

Comment:
Rather give a hint as to what _exact_ charset and encoding is needed, you'll just get queries on t/pootle to ask exactly that.